packages/core/services/FileService/HttpFileService/index.ts

```diff
@@ -110,6 +110,31 @@ export default class HttpFileService extends HttpServiceBase implements FileServ
         return response.data.map((file) => new FileDetail(file, env));
     }
 
+    private async getSelectionsCsv(
+        annotations: string[],
+        selections: Selection[]
+    ): Promise<{ url: string; data: Blob }> {
+        const postData = JSON.stringify({ annotations, selections });
+        const url = `${this.fileExplorerServiceBaseUrl}/${HttpFileService.BASE_CSV_DOWNLOAD_URL}${this.pathSuffix}`;
```

Comment:
could probably move this to config too

Reply:
This is moved from existing code below, I think `fileExplorerServiceBaseUrl` is selected from constants?